portability-transfer/src/main/java/org/datatransferproject/transfer/JobProcessor.java

```diff
@@ -153,6 +148,11 @@ void processJob() {
     } catch (IOException | CopyException | RuntimeException e) {
       monitor.severe(() -> "Error processing jobId: " + jobId, e, EventCode.WORKER_JOB_ERRORED);
     } finally {
+      final Collection<ErrorDetail> errors = copier.getErrors(jobId);
+      final int numErrors = errors.size();
+      monitor.debug(
+          () -> format("Finished copy for jobId: %s with %d error(s).", jobId, numErrors));
+      success &= errors.isEmpty();
       monitor.debug(() -> "Finished processing jobId: " + jobId, EventCode.WORKER_JOB_FINISHED);
```

Comment:
should we get rid of this existing debug line? or we wanted to keep two of them on purpose?

Reply:
Combined debug messages.